configure.ac

```diff
@@ -2492,6 +2492,20 @@ AC_DEFUN([PY_CHECK_CC_WARNING], [
   AS_VAR_POPDEF([py_var])
 ])
 
+dnl PY_CHECK_CC_CFLAG(ENABLE, CFLAG, [MSG])
```

Comment:
Is the first parameter needed?

Reply:
I am not very sure how this autoconf macros are working, but I borrowed the logic from  `PY_CHECK_CC_WARNING` which exists directly above.